src/pages/blog/starlingx-deploying-your-app-3.md

```diff
@@ -0,0 +1,306 @@
+Troubleshooting a large system like StarlingX can be complex and may require various approaches. This section provides a brief overview of useful commands I used to address common problems that arose during the installation of StarlingX applications.
+### sysinv log
+
+Usually, after seeing that the application has failed, this is probably the best first place for you to check out and find out more about what has gone wrong.
+The sysinv log is located on:
+```
+/var/log/sysinv.log
+```
+It may be useful using:
+```
+cat /var/log/sysinv.log | grep "application name"
+```
+
+### system application commands
+These commands may be beneficial for finding out about the state of an application when it did not deploy correctly, and you want to reinstall or manage it. Some commands are:
+* ```system application-list ```
+	- this command shows the list of applications uploaded or applied to the cluster and some top-level informations. Particularly, status and progress are useful for debbuging.
+* ```system application-show "application_name"```
+	- this command provides a most of the information provided in the _application-list_, but with extra rows: ["Active", "created_at", "name", "updated_at"].
+* ```system application-remove```
+	- this command removes an application from service. Removing an application will clean up related Kubernetes resources and delete all of its installed helm charts
+* ```system application-delete```
+	- we can use the following command to completely delete an application from the system **after running the system application-remove**
+
+more useful [system application commands](https://docs.starlingx.io/system_configuration/kubernetes/application-commands-and-helm-overrides.html)
+
+[bug when trying to remove and delete an application](https://wiki.openstack.org/wiki/StarlingX/Containers/FAQ#What_to_I_do_if_I_can.27t_re-run_an_application_action.3F)
+### kubernetes pod logs
+
+In some cases, You'll see in the sysinv.log that the application of a pod was created, but for some reason It is not working properly. In those cases, It is useful to check the Kubernetes log of that pod. The basic command is:
+```
+kubectl logs podname -n namespace
+```
+Also note that that information like the prefix of the pod and the namespace where It will be created and running are written in the helm chart of the application.
+
+**Please note** that troubleshooting can get more complicated than this, but I think these are useful commands that help to address and identify common installation problems.
+
+# Conclusion 
+With this post, we finish our StarlingX Application Deployment series consisting of three blog posts. If you are interested, check out the [blog post 1](https://www.starlingx.io/blog/starlingx-deploying-your-app-1) and the [blog post 2](https://www.starlingx.io/blog/starlingx-deploying-your-app-2/). If you have any suggestion or comment about the post, fell free to contact me through my e-mail tomas.barros@encora.com
```

Comment:
```suggestion
This post concludes the StarlingX Application Deployment [series](https://www.starlingx.io/blog/starlingx-deploying-your-app-1) consisting of three articles. If you have any questions, or need help with StarlingX, please reach out to the community on the [starlingx-discuss mailing list](https://lists.starlingx.io/mailman3/lists/starlingx-discuss.lists.starlingx.io/).
```

Reply:
In order to keep communication open, we direct people to the community's most active communication channels from the blog posts, which for StarlingX is the mailing list.

Reply:
sure. Thank you!